foreman/foreman.spec

```diff
@@ -719,7 +719,9 @@ sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/se
 export BUNDLER_EXT_GROUPS="default assets"
 ln -s %{nodejs_sitelib} node_modules
 export NODE_ENV=production
+%{?scl:scl enable %{scl} "}
 webpack.js --bail --config config/webpack.config.js
```

Comment:
@mmoll the script itself does not run under rails/rake it simply need access to bundler gem in order to find all plugin paths.